Modules/Microsoft365DSC/DSCResources/MSFT_AADAuthenticationMethodPolicyFido2/MSFT_AADAuthenticationMethodPolicyFido2.schema.mof

```diff
@@ -0,0 +1,38 @@
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphFido2KeyRestrictions
+{
+    [Write, Description("A collection of Authenticator Attestation GUIDs. AADGUIDs define key types and manufacturers.")] String AaGuids[];
+    [Write, Description("Enforcement type. Possible values are: allow, block."), ValueMap{"allow","block","unknownFutureValue"}, Values{"allow","block","unknownFutureValue"}] String EnforcementType;
+    [Write, Description("Determines if the configured key enforcement is enabled.")] Boolean IsEnforced;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphExcludeTarget2
```

Comment:
To avoid conflicts across resources in the future, we took the decision to always rename these CIMInstances as:

MSFT_resourceNameCIMInstance

Please review every schema.mof and module files in this Pull Request to adhere to this naming standard. Thanks